src/Microsoft.TemplateEngine.Edge/Settings/TemplatePackageManager.cs

```diff
@@ -313,8 +328,16 @@ private async Task<TemplateCache> UpdateTemplateCacheAsync(bool needsRebuild, Ca
             {
                 scanResult?.Dispose();
             }
-            JObject serialized = JObject.FromObject(cache);
-            _paths.WriteAllText(_paths.TemplateCacheFile, serialized.ToString());
+
```

Comment:
same story as for `ReadAllText` - we should benefit from this change everywhere.

Reply:
Created new JExtension extension method and call from everywhere...